raiden/blockchain/events.py

```diff
@@ -145,49 +144,53 @@ def get_all_netting_channel_events(
     )
 
 
-def decode_event_to_internal(event):
-    """ Enforce the binary encoding of address for internal usage. """
-    data = event.event_data
+def decode_event_to_internal(abi, log_event):
+    """ Enforce the binary for internal usage. """
+    # Note: All addresses inside the event_data must be decoded.
 
-    if data['args'].get('channel_identifier'):
-        data['channel_identifier'] = data['args'].get('channel_identifier')
```

Comment:
copying the id was completely unnecessary since `data['args']` is also available for the event handlers.